src/index.html

```diff
@@ -12,7 +12,79 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <div aria-label="Star holder">
```

Comment:
In this case, it would be more semantic to use the `<main>` tag instead of `<div>`. The `<main>` tag is used to wrap the main content of the body of a document or application. The content inside `<main>` should be unique to the document, excluding content that is repeated across a set of documents such as site navigation links, header or footer information [Use semantic tags where possible].